oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -81,20 +91,28 @@ impl Runtime {
     /// Thread safe method for signaling termination to a `Runtime` and waiting for its node
     /// threads to terminate.
     pub fn stop(&self) {
-        let handles = {
-            let mut node_threads = self.node_threads.lock().unwrap();
+        {
+            let nodes = self.nodes.lock().unwrap();
+            debug!("nodes: {:?}", nodes);
```

Comment:
Is this intended? Why not just move this after line 99?

Reply:
Removed: left over from checking things were working as intended :)